client/v2/udp.go

```diff
@@ -107,6 +108,10 @@ func (uc *udpclient) Query(q Query) (*Response, error) {
 	return nil, fmt.Errorf("Querying via UDP is not supported")
 }
 
+func (uc *udpclient) QueryCtx(ctx context.Context, q Query) (*Response, error) {
+	return nil, fmt.Errorf("Querying via UDP is not supported")
```

Comment:
Error messages in Go start with a lower case letter. `"querying via UDP ...`

Reply:
You are absolutly right. But I thought it would be best to keep the error message like the existing ones:
https://github.com/influxdata/influxdb/blob/71ac0134162d440702ced4824e0bd365ff540b2f/client/v2/udp.go#L108
https://github.com/influxdata/influxdb/blob/71ac0134162d440702ced4824e0bd365ff540b2f/client/v2/udp.go#L116